testing/rkt/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Brian Rosner <brian@brosner.com>
+# Maintainer: Brian Rosner <brian@brosner.com>
+pkgname=rkt
+pkgver=1.20.0
+pkgrel=0
+pkgdesc="container engine for Linux designed to be composable, secure, and built on standards"
+url="https://github.com/coreos/rkt"
+arch="x86_64"
+license="ASL 2.0"
+depends=""
+makedepends="$depends_dev coreutils grep gzip diffutils cpio go bc
+	linux-headers wget file gnupg autoconf automake bash libressl-dev
+	acl-dev xz sed"
+subpackages=""
+source="rkt-$pkgver.tar.gz::https://github.com/coreos/rkt/archive/v$pkgver.tar.gz"
+builddir="$srcdir/rkt-$pkgver"
+
+build() {
+	cd "$builddir"
+	./autogen.sh || return 1
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+		--enable-tpm=auto \
+		--enable-sdjournal=auto \
+		|| return 1
+	make || return 1
+}
+
+package() {
+	cd "$builddir"
+	install -Dm755 "build-rkt-$pkgver"/target/bin/rkt "$pkgdir"/usr/bin/rkt
+	local flavor; for flavor in fly coreos kvm; do
```

Comment:
I looked into what these “flavours” are and I think that it’d make sense to separate them into subpackages. What do you think?